edb/lib/schema.edgeql

```diff
@@ -76,7 +76,8 @@ CREATE TYPE schema::PseudoType EXTENDING schema::Type;
 
 ALTER TYPE schema::Type {
     CREATE PROPERTY expr -> std::str;
-    CREATE PROPERTY from_alias := EXISTS(.expr);
+    CREATE PROPERTY expr_type -> int64;
```

Comment:
Do we really need to expose `expr_type`?

Reply:
That is where `is_view` is computed from in the server code, so it's the main source of truth for this. 

I think it would also work to expose `alias_is_persistent`?

Or we could add a new `Field`

Reply:
What's `expr_type`? Why is it an int?

Reply:
It's an enum that indicates what kind of statement (select, insert, update, etc) a type is based on. The actual value is not really important here, but its presence is currently the main source of truth for whether a type is a view or not.

I'm going to rename `alias_is_persistent` to `is_persistent_alias` and expose that instead

Reply:
Or maybe just add a `from_alias` field?

Reply:
Can we call it `is_alias`?

Reply:
We don't use `is_` prefixes in schema anymore.